src/main.cpp

```diff
@@ -190,20 +209,57 @@ int T_main(int argc, TCHAR* argv[])
 		img = &exe;
 	else
 	{
-		// try DBG file alongside executable
-		changeExtension(dbgname, argv[1], TEXT(".dbg"));
 		struct _stat buffer;
+
+		if (debug_link || exe.hasDebugLink())

```

Comment:
Can this part be extracted to some function?

Reply:
Yeah, I meant the whole block.

Reply:
Done